src/Poseidon/EntitiesList.hs

```diff
@@ -13,10 +13,13 @@ import           Poseidon.Utils          (PoseidonException (..))
 
 import           Control.Applicative     ((<|>))
 import           Control.Exception       (throwIO)
+import           Data.Aeson              (FromJSON(..), ToJSON(..), withText, Value(..))
+import Data.Aeson.Types (Parser)
```

Comment:
We should probably switch to some sort of linter or beautifier to avoid inconsistent formatting.

Reply:
Yeah, I use stylish-haskell from vscode. Have just fixed this layout.